src/Compression/LZ4_decompress_faster.cpp

```diff
@@ -81,7 +81,7 @@ inline void copyOverlap8(UInt8 * op, const UInt8 *& match, size_t offset)
 }
 
 
-#if defined(__x86_64__) || defined(__PPC__) || defined(__riscv)
+#if defined(__x86_64__) || defined(__PPC__) || defined(__riscv) || defined(__s390x__)
```

Comment:
Cosmetic: could we keep the sort order used elsewhere: x86, ARM, PPC, s390x, riscv?